### PR TITLE
[FIX] Server assisted import for custom tokens

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

File: packages/bitcore-wallet-client/test/api.test.js [hunk 1]
```diff
@@ -6526,6 +6526,37 @@ describe('client API', function() {
...
+              Client.serverAssistedImport(
+              { words },
+              {
+                clientFactory: () => {
+                  return helpers.newClient(app);
+                }
+              },
+              (err, k, c) => {
+                // the eth wallet + 1 unknown token addresses on preferences.
+                c.length.should.equal(1);
```
Comment (kajoseph) [inline packages/bitcore-wallet-client/test/api.test.js:6546]: Should this be 2? The comment above makes me think it should be 2, but the oneInch API doesn't have a match for the tokenAddresses passed in preferences.

Comment (gabrielbazan7) [reply packages/bitcore-wallet-client/test/api.test.js:6546]: No, one is ok. I'm just testing that if there is an address it does not exist in the 1inch api retrieves the eth wallet without issues

**Review by kajoseph — APPROVED** [review]
LGTM

**Review by Gamboster — COMMENTED** [review]
ACK. Tested on my local BWS

**Pull request merged**